source/api_documentation/transactions_api/_signal_param_table.rst

```diff
@@ -53,4 +53,3 @@ Most of the fields in this table are now deprecated. See Custom Data & Signal Pa
   * - signal_custom_parameter_3 *(deprecated)*
     - Signal Custom Param 3
     - Up to 255 character string. |rollup_removed_message|
-
```

Comment:
I believe trailing new lines are needed, this should be undone

Reply:
If you look at the actual file there's a trailing newline, I'm removing an extra newline. Github's UI doesn't show the trailing line by default. If the file is missing a trailing newline github's UI will show a red return arrow at the bottom
e.g.
![image](https://user-images.githubusercontent.com/9993068/97094842-b14e0b80-160d-11eb-89be-0029aa63adde.png)

Reply:
🤦‍♂️